docs/application/native/guides/security/privilege.md

```diff
@@ -81,6 +81,22 @@ To get various privilege information:
                                                           &status);
     ```
 
+- Get the privilege information in a list form using the `privilege_info_get_privilege_info_list()` function:
+
+    ```
+    GList* privilege_name_list = NULL;
+    GList* privilege_info_list = NULL;
+    privilege_consumer_return_code_e return_result;
+
+    privilege_name_list = g_list_append(privilege_name_list, "http://tizen.org/privilege/call");
+    privilege_name_list = g_list_append(privilege_name_list, "http://tizen.org/privilege/contact.read");
+
+    int ret = privilege_info_get_privilege_info_list("ko_KR.UTF8",
+                                                     privilege_name_list,
+                                                     &privilege_info_list,
+                                                     &return_result);
+    ```
+
 The above functions return a `privilege_info_error_e` enum value (in [mobile](../../api/mobile/latest/group__CAPI__SECURITY__FRAMEWORK__PRIVILEGE__INFO__MODULE.html#gae50b814d4efe1b1d7218b6d68cdcadd6) and [wearable](../../api/wearable/latest/group__CAPI__SECURITY__FRAMEWORK__PRIVILEGE__INFO__MODULE.html#gae50b814d4efe1b1d7218b6d68cdcadd6) applications) that indicates the value retrieval result. They also store the requested privilege display name or description, privacy display, and privacy privilege status in their last parameter.
```

Comment:
These functions return a `privilege_info_error_e` enum value (in [mobile](../../api/mobile/latest/group__CAPI__SECURITY__FRAMEWORK__PRIVILEGE__INFO__MODULE.html#gae50b814d4efe1b1d7218b6d68cdcadd6) and [wearable](../../api/wearable/latest/group__CAPI__SECURITY__FRAMEWORK__PRIVILEGE__INFO__MODULE.html#gae50b814d4efe1b1d7218b6d68cdcadd6) applications) that indicates the value retrieval result. These functions also store the requested privilege display name or description, privacy display, and privacy privilege status in their last parameter.

Reply:
@yl33  kindly incorporate the suggested edits.

Reply:
Done.

Reply:
@yl33 thanks

Reply:
@yl33 I am unable to see the changes, please commit your changes.